services/app-web/src/pages/StateSubmission/ContractDetails/ContractDetailsSchema.ts

```diff
@@ -57,22 +62,21 @@ export const ContractDetailsFormSchema = Yup.object().shape({
         is: (contractType: string | undefined) => contractType,
         then: Yup.array().min(1, 'You must select at least one authority'),
     }),
-    itemsAmended: Yup.array().when('contractType', {
-        is: 'AMENDMENT',
-        then: Yup.array().min(1, 'You must select at least one item'),
-    }),
-    otherItemAmended: Yup.string().when('itemsAmended', {
-        is: (items: string[]) => items.includes('OTHER'),
-        then: Yup.string().defined('You must enter a description'),
-    }),
-    capitationRates: Yup.string().when('itemsAmended', {
-        is: (items: string[]) => items.includes('CAPITATION_RATES'),
-        then: Yup.string()
-            .nullable()
-            .defined('You must select a reason for capitation rate change'),
-    }),
-    capitationRatesOther: Yup.string().when('capitationRates', {
-        is: 'OTHER',
-        then: Yup.string().defined('You must enter a description'),
-    }),
+
+    modifiedBenefitsProvided: yesNoError,
```

Comment:
Nice. We should probably do this more often, creating re-useable yup validation types